app.rb

```diff
@@ -2,10 +2,10 @@
 
 require_relative 'lib/state_manager'
 require_relative 'lib/query_builder'
-require_relative 'lib/pg_manager'
+require_relative 'lib/sierra_db_manager'
```

Comment:
Is it a "manager" or a "client"? The class name has `Client`, which I prefer personally.

Reply:
Ah good catch. Fixed